include/kernel/paging.h

```diff
@@ -0,0 +1,24 @@
+// SPDX-FileCopyrightText: CGL-KFS
+// SPDX-License-Identifier: BSD-3-Clause
+
+/* include/kernel/paging.h
```

Comment:
Shouldn't this file be named `kvm.h` as we discussed earlier?

Reply:
No